examples/apps/ai-collab/src/app/presence.ts

```diff
@@ -5,24 +5,23 @@
 
 import {
 	IPresence,
-	LatestMap,
+	Latest,
 	type PresenceStates,
 	type PresenceStatesSchema,
 } from "@fluidframework/presence/alpha";
-import { v4 as uuid } from "uuid";
 
 export interface User {
 	photo: string;
 }
 
 const statesSchema = {
-	onlineUsers: LatestMap<{ value: User }, `id-${string}`>(),
+	onlineUsers: Latest<User>({ photo: "" }),
```

Comment:
Is empty string special for the `photo` property? Can we document why we're setting it that way?